dataScripts/tessTrain/tessTrain.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+set -e
+
+# Tested on a bare install of Ubuntu 22.04.3 LTS - 02/27/2024
+# Author: Charles@CharlesBock.com - github.com/Penguin2600
+
+# Color Stuff
+BGreen='\033[1;32m'
+NC='\033[0m'
+
+greentext () {
+   echo -e "\n${BGreen}### $1 ${NC}\n"
+}
+
+# Build tesseract
+# Upstream Docs: https://tesseract-ocr.github.io/tessdoc/Compiling-%E2%80%93-GitInstallation#installing-with-autoconf-tools
+greentext "Installing Deps and Creating File Structure"
+
+# Dont polute the directory
+mkdir -p ./tess
```

Comment:
Since this script creates artifacts, we'll need to add them to a `.gitignore` file. Ideally, we would keep the sole `.gitignore` so it's consolidated in one place.

Reply:
Good Callout, I'll consider that the new entries might need to be.